plexpy/notifiers.py

```diff
@@ -2721,3 +2748,94 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+class HIPCHAT(object):
+
+    def __init__(self):
+        self.apiurl = plexpy.CONFIG.HIPCHAT_URL
+        self.color = plexpy.CONFIG.HIPCHAT_COLOR
+        self.incl_subject = plexpy.CONFIG.HIPCHAT_INCL_SUBJECT
+        self.emoticon = plexpy.CONFIG.HIPCHAT_EMOTICON
+
+    def notify(self, message, subject):
+        if not message or not subject:
+            return
+        
+        if self.incl_subject:
+            text = subject.encode('utf-8') + ': ' + message.encode('utf-8')
+        else:
+            text = message.encode('utf-8')
+
+        if self.emoticon:
+            text = self.emoticon + ' ' + text
+
+        data = {'color': self.color,
```

Comment:
If I send a json POST with a blank color value it gives an error, but if the color option is not sent at all, it works fine and uses grey.  Examples of output colors:
![background](https://cloud.githubusercontent.com/assets/605448/16700319/6744d4e4-4527-11e6-896d-399339c1c90b.png)